### PR TITLE
Style guide

### PR DESCRIPTION
**Pull request by UX-Chris**
Description
-----------
Add a Astrum style guide with color, typography, navigation and text input components.

Motivation and Context
----------------------
To add a consistent style to the project. This addresses the issue:
https://github.com/request-yo-racks/web/issues/8

How Has This Been Tested?
-------------------------

Screenshots:
------------

image

Types of changes
----------------

- New feature (non-breaking change which adds functionality)

Checklist:
----------

-  [] I have updated the documentation accordingly.
-  [] I have written unit tests

Fixes: https://github.com/request-yo-racks/web/8

### CODE REVIEW

File: astrum-config.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "path": "./public/pattern-library"
+}
```
Comment (rgreinho) [inline astrum-config.json]: Add a new line at the end of file.

File: public/pattern-library/data.json [hunk 2]
```diff
@@ -0,0 +1,128 @@
+{
+    "project_logo": null,
+    "project_favicon": "ryr_logo.png",
```
Comment (rgreinho) [inline public/pattern-library/data.json]: I think the logo should also be set.

File: public/pattern-library/data.json [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    "project_name": "Request Yo Racks",
+    "project_url": null,
```
Comment (rgreinho) [inline public/pattern-library/data.json]: You can put the github url until we decide to publish something: http://request-yo-racks.github.io/.

File: public/pattern-library/data.json (hunk 2 above)
Comment (UX-Chris) [reply public/pattern-library/data.json]: I set the logo as the same image but I'm thinking we should change the favicon to the logo without the Request Yo Racks on the bottom.

Comment (rgreinho) [reply public/pattern-library/data.json]: Yep, totally agree.

**Pull request merged**